.github/workflows/focal_postgres12_postgis3.yml

```diff
@@ -0,0 +1,28 @@
+name: "[ubuntu-20.04] PostgreSQL 12 and PostGIS 3"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: Building and testing
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install PostgreSQL and PostGIS
+        env:
+          POSTGRESQL_VERSION: 12
+          POSTGIS_VERSION: 3
+        run: .github/scripts/postgresql_postgis.sh
+      - name: Install and check PgPointCloud
+        env:
+          POSTGRESQL_VERSION: 12
+          POSTGIS_VERSION: 3
```

Comment:
3 is for 3.0, 3.1 or 3.2?

Reply:
With these environment variables the next package is installed: `postgresql-$POSTGRESQL_VERSION-postgis-$POSTGIS_VERSION`. So in this case it's 3.2.

Reply:
(i'm going o update the tag and the doc accordingly)